src/Framework/Framework/Controls/HtmlGenericControl.cs

```diff
@@ -597,11 +598,11 @@ public sealed record HtmlCapability
             Visible.HasValue && Visible.ValueOrDefault == true &&
             ID.HasValue && ID.ValueOrDefault == null;
 
-        /// <summary> A dictionary of css styles which will be placed in the `style` attribute. </summary>
+        /// <summary> Gets or sets a dictionary of CSS styles which will be placed in the `style` attribute. </summary>
         [PropertyGroup("Style-")]
         public IDictionary<string, ValueOrBinding<object?>> CssStyles { get; init; } = new Dictionary<string, ValueOrBinding<object?>>();
 
-        /// <summary> When set to false, `style="display: none"` will be added to this control. </summary>
+        /// <summary> Gets or sets whether the control is visible. When set to false, `style="display: none"` will be added to this control. </summary>
```

Comment:
I'm not a fan of adding fluff to the comments. It's obvious that has getter, setter and controls if the control is visible (actually the comment is even partially wrong about the setter...)